src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/ReferenceBases.java

```diff
@@ -22,14 +23,19 @@
 
     public static final int NUM_BASES_ON_EITHER_SIDE = 10;
 
+    protected final OneShotLogger warning = new OneShotLogger(this.getClass());
+
     @Override
     public List<String> getKeyNames() { return Collections.singletonList(REFERENCE_BASES_KEY); }
 
     @Override
     public Map<String, Object> annotate(final ReferenceContext ref,
                                         final VariantContext vc,
                                         final ReadLikelihoods<Allele> likelihoods) {
-
+        if (ref==null)  {
```

Comment:
I though that https://github.com/broadinstitute/gatk/issues/2799 suggest to throw an `UserException`, which actually makes sense if the user provides an annotation that couldn't be apply because the requirements are not met...